src/discord.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const discordjs = require('discord.js')
+const getRepo = require('./parser').getRepoName
+
+var client = new discordjs.Client()
+var token = process.env[`npm_package_config_token`]
+var notify = process.env[`npm_package_config_discordnotify`]
+var defaultNotify = process.env[`npm_package_config_discorddefault`]
+
+if (notify) notify = JSON.parse(notify)
```

Comment:
This should be a `Map` to avoid issues with prototypes. Then use `get` API instead of indexing.

```suggestion
if (notify) notify = new Map(Object.entries(JSON.parse(notify)))
```